packages/server/postgres/queries/updateTeamByOrgId.ts

```diff
@@ -9,7 +9,7 @@ const updateTeamByOrgId = async (
   teamUpdates: Partial<IUpdateTeamByOrgIdQueryParams>,
   orgId: string
 ) => {
-  await catchAndLog(() =>
+  return await catchAndLog(() =>
```

Comment:
no `return await` (i'll turn on a rule after we get this merged)
remove catchAndLog.